class Main {
  public static void main(String[] args) {

    try {
      int divideByZero = 100 / 0;
      System.out.println("exception ");
    }
    
    catch (ArithmeticException e) {
      System.out.println("ArithmeticException => " + e.getMessage());
    }
  }
}
